src/discordtipbot/tipbot.cr

```diff
@@ -132,7 +132,7 @@ class TipBot
   end
 
   def db_balance
-    @db.query_one("SELECT SUM (balance) FROM accounts", &.read(Float64))
+    @db.query_one("SELECT SUM (balance) FROM accounts", &.read(BigDecimal))
```

Comment:
We should be using that other `query_one` overload.
```cr
def query_one(query, *args, as type : Class)
```
<http://crystal-lang.github.io/crystal-db/api/0.5.0/DB/QueryMethods.html#query_one%28query%2C%2Aargs%2Castype%3AClass%29-instance-method>